chalice/test.py

```diff
@@ -323,6 +323,29 @@ def generate_kinesis_event(self, message_bodies,
         } for body in message_bodies]
         return {'Records': records}
 
+    def generate_dynamodb_event(self, images):
+        # type: (List[Tuple[Dict, Dict]]) -> Dict[str, Any]
+        records = [{
+            "dynamodb": {
+                "ApproximateCreationDateTime": 1545084650.987,
+                "Keys": list(new_image.keys()),
+                "NewImage": new_image,
```

Comment:
Are the `NewImage`/`OldImage` keys `null` or omitted if your view type doesn't include them, e.g. `KEYS_ONLY`?  We should try and match that behavior.

Reply:
It's not currently respecting the view type properly, will update the behaviour so it omits the relevant keys for the `KEYS_ONLY`/`NEW_IMAGE`/`OLD_IMAGE`  stream view types